ansible/inventories/devnet-0/group_vars/all/images.yaml

```diff
@@ -1,30 +1,33 @@
 default_ethereum_client_images:
-  # Consensus layer clients
-  prysm: gcr.io/prysmaticlabs/prysm/beacon-chain:v4.0.7
-  prysm_validator: gcr.io/prysmaticlabs/prysm/validator:v4.0.7
-  teku: consensys/teku:23.6.2-amd64
-  lighthouse: sigp/lighthouse:v4.3.0
-  nimbus: statusim/nimbus-eth2:multiarch-v23.7.0
-  lodestar: chainsafe/lodestar:v1.9.2
-  # Execution layer clients
-  geth: ethereum/client-go:v1.12.0
-  nethermind: nethermind/nethermind:1.20.1
-  besu: hyperledger/besu:23.4.4-amd64
-  erigon: ethpandaops/erigon:latest
-  ethereumjs: g11tech/ethereumjs:latest
+### Consensus layer clients
```

Comment:
Ser no, template testnet has nothing to do with 4844 and we should not be using devnets as defaults. We should always use the latest stable release as that's the only sane default at this abstraction level